### PR TITLE
Add dutch, spanish all variants of english and allow running from _dev

### PR DESCRIPTION
**Pull request by barrydegraaff**
Hello Dennis,

New PR:
Add dutch, spanish all variants of english and allow running from _dev to avoid cache issues and to allow easy debugging on ZCS upgrades. 

Thanks Barry

### CODE REVIEW

File: com_zimbra_attachmentalert/attachmentalert.js [hunk 1]
```diff
@@ -63,8 +63,8 @@ AttachAlertZimlet.prototype._initializeRegEx =
 function() {
 	if (this._attachWordsRegEx)
 		return;
-	this._attachStr = this.getMessage("AttachmentAlert_attach");
-	this._errorMsgStr = this.getMessage("AttachmentAlert_error_noattachment");
+	this._attachStr = (this.getMessage("AttachmentAlert_attach").indexOf('???')>=0 ? 'attach,bijlage' : this.getMessage("AttachmentAlert_attach"));
```
Comment (dploeger) [inline com_zimbra_attachmentalert/attachmentalert.js]: What's the purpose of this?

File: com_zimbra_attachmentalert/com_zimbra_attachmentalert.properties [hunk 2]
```diff
@@ -19,7 +19,7 @@ description = Alerts you when forgetting to attach an attachment when composing
 #
 # Defines the attach search strings (a comma separated list)
 #
-AttachmentAlert_attach = attach
+AttachmentAlert_attach = attach,bijlage
```
Comment (dploeger) [inline com_zimbra_attachmentalert/com_zimbra_attachmentalert.properties]: Why do you change the default (english) localization files?

**Comment by dploeger** [comment]
Hi David!

Did you change that because of the debugging purposes? Sorry, I can't accept that, because that would contaminate the usual localization features. 

You can still run the zimlet from _dev by issuing a "zmprov fc uistrings" after deploying.

Wouldn't that work?

Thanks,
Dennis

File: com_zimbra_attachmentalert/com_zimbra_attachmentalert.properties (hunk 2 above)
Comment (barrydegraaff) [inline com_zimbra_attachmentalert/com_zimbra_attachmentalert.properties]: Cause this Zimlet only works for 1 language at the time. My users have Zimbra set to the default language english, and still expect to be warned if the missed an attachment while typing a dutch email.

Maybe I should fork this zimlet and make a 'Configurable version'...

**Comment by barrydegraaff** [comment]
zmprov fc uistrings or zmprov fc all don't work

https://bugzilla.zimbra.com/show_bug.cgi?id=97708
Of course this will never gets fixed.

I prefer to be able to run from _dev folder, to avoid cache issues and to be able to make changes appear fast.

In _dev folder this.getMessage will return strings like ???message_id???, I use additional if statements to see if I get ??? and then return the default english texts. If deployed from zip, 
the usual translation just works in Zimbra.

Barry

2015-07-09-081833_1920x1200_scrot

File: com_zimbra_attachmentalert/attachmentalert.js (hunk 1 above)
Comment (barrydegraaff) [inline com_zimbra_attachmentalert/attachmentalert.js]: In _dev folder this.getMessage will return strings like ???message_id???, I use additional if statements to see if I get ??? and then return the default english texts. If deployed from zip, 
the usual translation just works in Zimbra.

**Comment by dploeger** [comment]
Hm? I'm well sure, that this works, as I have always developed that way. I'll check this and get back to you.

**Comment by barrydegraaff** [comment]
Okay, I'm interested in that solution, in the mean time, I will make a clean PR

**Pull request closed without merging**